[Linux/Mesa] 6_deferred_async_load error: "transfer_queue := gpu.get_queue(.Transfer)" results in "[gpu_vulkan.odin:2595:vk_submit_cmd_buf()] Vulkan failure: ERROR_DEVICE_LOST"

Getting this error on Linux running latest Mesa git Main. (3_3D runs perfectly fine without errors, by comparison.)
```
[ERROR] --- [2026-01-23 13:49:20] [gpu_vulkan.odin:2252:vk_debug_callback()] vkCreateDevice(): pCreateInfo->pQueueCreateInfos[0].queueCount (2) is not less than or equal to available queue count for this pCreateInfo->pQueueCreateInfos[0].queueFamilyIndex (0) obtained previously from vkGetPhysicalDeviceQueueFamilyProperties or vkGetPhysicalDeviceQueueFamilyProperties2[KHR] (needs to be less than or equal to 1).
The Vulkan spec states: queueCount must be less than or equal to the queueCount member of the VkQueueFamilyProperties structure, as returned by vkGetPhysicalDeviceQueueFamilyProperties in the pQueueFamilyProperties[queueFamilyIndex] (https://docs.vulkan.org/spec/latest/chapters/devsandqueues.html#VUID-VkDeviceQueueCreateInfo-queueCount-00382)
[INFO ] --- [2026-01-23 13:49:20] [shared.odin:348:load_scene_gltf()] Collecting texture info from 69 textures in GLTF
[DEBUG] --- [2026-01-23 13:49:20] [main.odin:210:texture_loader_thread_proc()] Creating texture loader for chunk 0 of 307
[DEBUG] --- [2026-01-23 13:49:20] [main.odin:210:texture_loader_thread_proc()] Creating texture loader for chunk 16 of 307
radv/amdgpu: The CS has been rejected, see dmesg for more information (-22).
[FATAL] --- [2026-01-23 13:49:20] [gpu_vulkan.odin:2596:vk_submit_cmd_buf()] Vulkan failure: ERROR_DEVICE_LOST
/run/media/valmar/IronWolfRAID1/Repositories/Odin/no_gfx_api/gpu/gpu_vulkan.odin(2262:9) panic
[1]    516940 illegal hardware instruction (core dumped)  odin run . -debug
```

dmesg output:
```
[275391.895282] traps: 6_deferred_asyn[501020] trap invalid opcode ip:489a45 sp:7f6b505e50e0 error:0 in 6_deferred_async_load[89a45,402000+11b000]
```

--- a/gpu/gpu_vulkan.odin
+++ b/gpu/gpu_vulkan.odin
@@ -353,8 +353,6 @@ _init :: proc()
         }
     }
 
-    log.debug("queue_create_infos:", queue_create_infos) 
-
     // Device
     device_extensions := []cstring {
         vk.KHR_SWAPCHAIN_EXTENSION_NAME,
@@ -425,8 +423,6 @@ _init :: proc()
     for &queue in ctx.queues[1:] {
         vk.GetDeviceQueue(ctx.device, queue.family_idx, queue.queue_idx, &queue.handle)
     }
-
-    log.debug("queues:", ctx.queues)
 
     // Common resources
     {

